Homarus/README.md

```diff
@@ -0,0 +1,26 @@
+# Homarus
+
+### Apache Config
+```
+# managed by Ansible
+
+Alias "/homarus" "/var/www/html/Crayfish/Homarus/src"
+<Directory "/var/www/html/Crayfish/Homarus/src">
+  FallbackResource /homarus/index.php
+  Require all granted
+  DirectoryIndex index.php
+  SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
+</Directory>
+```
+
+### Testing
+* Clone this microservice into `/var/www/html/Crayfish`
+* Install the depencies using composer `composer install`
+* Place the above confir in `/etc/apache2/conf-available` as `Homarus.conf`.  
+* Enable the conf: `sudo a2enconf Homarus`.  Restart Apache: `sudo systemctl restart apache2`
```

Comment:
* `sudo apt-get install ffmpeg`
* Copy `/var/www/html/Crayfish/Homarus/cfg/config.default.yml` to `/var/www/html/Crayfish/Homarus/cfg/config.yml`
* Copy `/var/www/html/Crayfish/Hypercube/syn-settings.xml` to `/var/www/html/Crayfish/Homarus/syn-settings.xml`